src/common/utils/custom-field-validation.ts

```diff
@@ -0,0 +1,32 @@
+export class CustomFieldsValidation {
+    static validate(fieldName: string, fieldValue: string) {
```

Comment:
You will validate field against the type and not name

    static validate(fieldType: string, fieldValue: string) {